Makefile

```diff
@@ -248,6 +248,7 @@ lint:
 	isort $(PYCOSM_SRC_DIR) $(PYCOSM_TESTS_DIR) $(PYCOSM_EXAMPLES_DIR) setup.py
 	flake8 $(PYCOSM_SRC_DIR) $(PYCOSM_TESTS_DIR) $(PYCOSM_EXAMPLES_DIR) setup.py
 	vulture $(PYCOSM_SRC_DIR) $(PYCOSM_TESTS_DIR) $(PYCOSM_EXAMPLES_DIR) setup.py --exclude '*_pb2.py,*_pb2_grpc.py'
+	bash -c "find $(PYCOSM_SRC_DIR) $(PYCOSM_TESTS_DIR) $(PYCOSM_EXAMPLES_DIR) -iname '*.py'|grep -v cosmpy/protos|xargs darglint"
```

Comment:
This is a bit messy. Couldn't you add it cleaner like in the aea framework https://github.com/fetchai/agents-aea/blob/main/Makefile?